src/evm_transition_tool/execution_specs.py

```diff
@@ -126,6 +74,8 @@ def start_server(self):
                 "--uds",
                 self.server_file_path,
             ],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
```

Comment:
Silences the post requests when we use `-s` during a fill.

This is a pytest flag that sets the stdout capture to false. Ideally we don't print the post request responses when filling with -s! Maybe there is a better solution than this though.

I mostly use it for fast print statement debugging.